docs/live_sessions.md

```diff
@@ -0,0 +1,27 @@
+# Live sessions
+
+Marsha provides a way to track student activity during a live. 
+It is done by creating a "live session" when the student starts watching the live 
+or registers to it.
+
+It slowly evolved to become a way to store the student attendance to a live and extract 
+an "attendance sheet" (a report telling who watched the live) for the instructor.
+
+The "live session" also stores the "display name" of the student which is displayed
+in the attendance sheet and the chat. The attendance sheet displays the student's display name and username.
+
+For the public lives, the "live session" can be defined as anonymous because the student
+is not required to be logged in to watch the live. In this case, the session has an `anonymous_id`
+and the authentication is made through a ResourceToken (ie LTI like) regardless of the context (LTI or standalone).
```

Comment:
```suggestion
and the authentication is made through a ResourceToken (ie LTI like) or a UserToken (standalone).
```
The context matter ! on the standalone site you will never had a `ResourceToken`.